addon/components/u-edit/shared-modules/drag-and-drop.js

```diff
@@ -0,0 +1,91 @@
+import Uploader from '@upfluence/ember-upf-utils/uploader';
+import Configuration from '@upfluence/ember-upf-utils/configuration';
+
+export default class ModuleBuilder {
+  constructor(session, toast) {
+    Object.assign(this, { session, toast });
+  }
+
+  build(editor, element) {
+    return new Module(editor, element, this.session.data.authenticated.access_token, this.toast);
```

Comment:
this is wrong IMO because this will never change in time. With not just pass the session service ? and fetch the access_token when needed ?